ITK failed to build due to error C2169: 'lrintf': intrinsic function, cannot be defined with MSVC on windows

Hi All,

ITK failed to build due to error C2169: 'lrintf': intrinsic function, cannot be defined with MSVC on windows. It can be reproduced on master branch latest commit 26079ef.  Could you please help look at this? Thanks a lot!

Repro Steps:
1. git clone https://github.com/InsightSoftwareConsortium/ITK.git F:\gitP\InsightSoftwareConsortium\ITK
2. Open VS2019 x64 command prompt as admin and browse to F:\gitP\InsightSoftwareConsortium\ITK
3. Apply itk_bug623130.patch
4. mkdir build_amd64 and cd build_amd64
5. cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_SYSTEM_VERSION=10.0.18362.0 -DBUILD_SHARED_LIBS=OFF -DBUILD_TESTING=ON -DBUILD_EXAMPLES=OFF ..
6. set _CL_=/D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING %_CL_%
7. msbuild /m /p:Platform=x64 /p:Configuration=Release All_BUILD.vcxproj /t:Rebuild

Build_log:
[build.log](https://github.com/InsightSoftwareConsortium/ITK/files/5141200/build.log)
itk_bug623130.patch:
[itk_bug623130.zip](https://github.com/InsightSoftwareConsortium/ITK/files/5141203/itk_bug623130.zip)

ErrorMessage:
188>F:\gitP\InsightSoftwareConsortium\ITK\Modules\ThirdParty\OpenJPEG\src\openjpeg\opj_includes.h(106,35): error C2169: 'lrintf': intrinsic function, cannot be defined [F:\gitP\InsightSoftwareConsortium\ITK\build_amd64\Modules\ThirdParty\OpenJPEG\src\openjpeg\itkopenjpeg.vcxproj]

--- a/Modules/ThirdParty/OpenJPEG/src/openjpeg/opj_includes.h
+++ b/Modules/ThirdParty/OpenJPEG/src/openjpeg/opj_includes.h
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2005, Hervé Drolon, FreeImage Team
+ * Copyright (c) 2005, HervÃ© Drolon, FreeImage Team
  * Copyright (c) 2008, Jerome Fimes, Communications & Systemes <jerome.fimes@c-s.fr>
  * All rights reserved.
  *
@@ -92,7 +92,7 @@ Most compilers implement their own version of this keyword ...
 
 /* MSVC 64bits doesn't support _asm */
 #if !defined(_WIN64)
-static INLINE long lrintf(float f){
+static INLINE long opj_lrintf(float f){
   int i;
 
   _asm{
@@ -103,7 +103,7 @@ static INLINE long lrintf(float f){
   return i;
 }
 #else
-static INLINE long lrintf(float x){
+static INLINE long opj_lrintf(float x){
   long r;
   if (x>=0.f)
   {

